### PR TITLE
Integrate the connection between ximalaya and exoplayer

### PR DESCRIPTION
**Pull request by ElenaWang666**

### CODE REVIEW

File: shared/src/main/java/bmw/ximalaya/test/extensions/XmlyMediaPlayer.kt [hunk 1]
```diff
@@ -48,15 +48,15 @@ class XmlyMediaPlayer(ctx: Context) : IXmPlayerStatusListener, IXmAdsStatusListe
...
-//        config.useProxy = false // 若想使用代理，必须配置此项为true，否则代理配置被忽略
-//
-//        config.proxyHost = "172.30.50.10"
-//        config.proxyPort = 8080
-//        config.connectionTimeOut = 3000
-//        config.readTimeOut = 3000
-//        config.writeTimeOut = 3000
-//        CommonRequest.getInstanse().httpConfig = config
+        val config: Config = Config()
+        config.useProxy = false // 若想使用代理，必须配置此项为true，否则代理配置被忽略
```
Comment (Lucifer9204) [inline shared/src/main/java/bmw/ximalaya/test/extensions/XmlyMediaPlayer.kt]: Remove or change the Chinese comment

File: shared/src/main/java/bmw/ximalaya/test/media/BrowseTree.kt [hunk 2]
```diff
@@ -1,76 +1,109 @@
...
                 }
-                mediaIdToChildren["id_jdlg"] = mutableListOf<MediaMetadataCompat>().apply{addAll(sourceList)}
-                this[NEU_HOME_ROOT]?.addAll(sourceList)
-                this[NEU_RECENT_ROOT]?.addAll(sourceList)
-                this[NEU_LIBRARY_ROOT]?.addAll(sourceList)
+
+                NeuLog.e("mediaItems(${musicSource.albumList})")
+                this[TINGYU_HOME_ROOT]?.addAll(
+                    musicSource.albumList
+//                    getMediaMetadataCompats()
```
Comment (Lucifer9204) [inline shared/src/main/java/bmw/ximalaya/test/media/BrowseTree.kt:85]: Remove the useless code

File: shared/src/main/java/bmw/ximalaya/test/extensions/XmlyMediaPlayer.kt (hunk 1 above)
Comment (ElenaWang666) [reply shared/src/main/java/bmw/ximalaya/test/extensions/XmlyMediaPlayer.kt]: Delete

File: shared/src/main/java/bmw/ximalaya/test/media/BrowseTree.kt (hunk 2 above)
Comment (ElenaWang666) [reply shared/src/main/java/bmw/ximalaya/test/media/BrowseTree.kt:85]: Need to keep during debugging, will be removed before official released.

**Pull request merged**